code-generation/generator/src/main/resources/com/amazonaws/util/awsclientgenerator/velocity/cpp/ModelClassMembersAndInlines.vm

```diff
@@ -63,16 +63,20 @@
 #set($setterWithCopy = " { ${required}${memberVariableName} = value; }")
 #set($setterWithMove = " { ${required}${memberVariableName} = std::move(value); }")
 #end
+#set($getterFunctionName = "Get${memberKeyWithFirstLetterCapitalized}")
+#if($getterFunctionName == ${otherMember.value.shape.name})
```

Comment:
This place is executed in a loop
in case of very huge models, it may affect the performance/increase the code gen time.
Please try to run ec2 code generation with and without change to see the diff.